gadget3.Rproj

```diff
@@ -0,0 +1,17 @@
+Version: 1.0
+
+RestoreWorkspace: Default
+SaveWorkspace: Default
+AlwaysSaveHistory: Default
+
+EnableCodeIndexing: Yes
+UseSpacesForTab: Yes
+NumSpacesForTab: 2
```

Comment:
This should be 4 if you're setting it. You should also set ``StripTrailingWhitespace: Yes`` (I think! There's no documentation for .Rproj files) to avoid the whitespace noise above.

Reply:
Do we want to have Rproj files in the repo?

Reply:
In general, configuration options to make an editor do the right thing would definitely make sense in the repo, rather than you having to change configuration manually (and work out what you need to change). But the workspace/history options feel too much like user preference, and I'd be nervous about setting them like some projects seem to.

It doesn't bother me either way, whichever makes your lives easier. If you're likely to spend more time fighting it then we leave it out.

Reply:
Exactly, these Rproj files are automaticaly generated by Rstudio so they are bound to cause problems even when the user did not intend to edit the file. I would opt for ignoring Rproj files.

Reply:
Ah...sorry. I didn't see .Rproj in the .gitignore, so I wasn't sure. I'll remove it.